### ISSUE TITLE
matedata error

### ISSUE THREAD

**Issue by slashui** [alfredonline/alfredonline-NextJS-MongoDB-Blog-Template#1]
How can I fix it? thank u
Error message:
ReactServerComponentsError:

You are attempting to export "metadata" from a component marked with "use client", which is disallowed. Either remove the export, or the "use client" directive. Read more: https://nextjs.org/docs/getting-started/react-essentials#the-use-client-directive

    ,-[/Users/slashui/Downloads/alfredonline-NextJS-MongoDB-Blog-Template-main/app/layout.js:7:1]
  7 | 
  8 | const inter = Inter({ subsets: ["latin"] });
  9 | 
 10 | export const metadata = {
    :              ^^^^^^^^
 11 |   title: "Alfred Web Dev Blog",
 12 |   description: "The best blog for web development",
 13 | };
    `----

File path:
  ./app/layout.js

**Comment by alfredonline** [comment]
Hello, did you convert it to a client component by adding "use client" at the top of the file?

**Comment by slashui** [comment]
Yes, built exactly to your code! If delect "use client". Errors will be reported elsewhere.

**Comment by slashui** [comment]
Another idea, how to skip the meta settings?

**Comment by slashui** [comment]
OK, just delete the metadata settings！ Thank you for the video!!
